.env.exmaple

```diff
@@ -0,0 +1,7 @@
+SD_BACKEND_URL="/v2"
+
+SD_CLIENT_ID=""
+SD_AUTHORITY=""
```

Comment:
Is this env variable for an authority url? If yes, please add `_URL` to the end.